### PR TITLE
[up-nrhm] exclude deleted users from Location Hierarchy

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @czue @emord 

We need exclude deleted users also in the up-nrhm domain. I talked today with Manish and the users are not deactivated but deleted. To don't have problems in future I exclude both options.

Need QA: no
Environment: production
Locally tested: yes

Regards,
Łukasz

### CODE REVIEW

File: custom/up_nrhm/data_sources/location_hierarchy.json [hunk 1]
```diff
@@ -7,15 +7,31 @@
...
+                    "type": "boolean_expression",
+                    "comment": null,
+                    "property_value": true
+                },
+                {
+                    "operator": "eq",
+                    "expression": {
+                         "datatype": null,
+                         "type": "property_name",
+                         "property_name": "-deletion_date"
```
Comment (emord) [inline custom/up_nrhm/data_sources/location_hierarchy.json]: Was it intentional to have the `-` at the start of this?

Comment (lwyszomi) [reply custom/up_nrhm/data_sources/location_hierarchy.json]: @emord we have that ```-``` in the property name
example_doc

**Review by emord — APPROVED** [review]

File: custom/up_nrhm/data_sources/location_hierarchy.json (hunk 1 above)
Comment (emord) [reply custom/up_nrhm/data_sources/location_hierarchy.json]: May be better to filter on `base_doc = "CouchUser-Deleted"` but this is fine .

Comment (mkangia) [reply custom/up_nrhm/data_sources/location_hierarchy.json]: Using base_doc = "CouchUser-Deleted" is more in sync with how HQ checks if a user is deleted, so would be good to use that .

Comment (lwyszomi) [reply custom/up_nrhm/data_sources/location_hierarchy.json]: @emord @mkangia updated

**Review by emord — APPROVED** [review]

File: custom/up_nrhm/data_sources/location_hierarchy.json (hunk 1 above)
Comment (mkangia) [reply custom/up_nrhm/data_sources/location_hierarchy.json]: thank you @lwyszomi

**Pull request merged**